instrumentation/github.com/aws/aws-lambda-go/otellambda/go.mod

```diff
@@ -0,0 +1,13 @@
+module go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda
+
+go 1.15
+
+replace go.opentelemetry.io/contrib => ../../../../..
+
+require (
+	github.com/aws/aws-lambda-go v1.24.0
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/contrib v0.22.0
+	go.opentelemetry.io/otel v1.0.0-RC2
```

Comment:
should these be updated to 1.0.0 or 1.0.1?

Reply:
Updated!